src/test/java/com/vaadin/testbenchexample/junit5/AbstractJUnit5IT.java

```diff
@@ -0,0 +1,47 @@
+package com.vaadin.testbenchexample.junit5;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+
+import com.vaadin.flow.component.button.testbench.ButtonElement;
+import com.vaadin.flow.component.textfield.testbench.TextFieldElement;
+import com.vaadin.testbench.BrowserTestBase;
+import com.vaadin.testbench.DriverSupplier;
+import com.vaadin.testbench.IPAddress;
+
+/**
+ * Base class for all our JUnit5 tests, allowing us to change the applicable driver,
+ * test URL or other configurations in one place.
+ */
+public abstract class AbstractJUnit5IT extends BrowserTestBase implements DriverSupplier {
+
+    @Override
+    public WebDriver createDriver() {
+        return new ChromeDriver();
+    }
+
+    @BeforeEach
+    public void setUp() {
+        getDriver().get("http://" + IPAddress.findSiteLocalAddress() + ":8080");
+    }
+
+    // IT test for demo purposes
+    protected void calculate_resultEquals(String a, String op, String b, String result) {
+        $(ButtonElement.class).id("button_" + a).click();
+        $(ButtonElement.class).id("button_" + op).click();
+        $(ButtonElement.class).id("button_" + b).click();
+        $(ButtonElement.class).id("button_=").click();
+        Assertions.assertEquals(result,
+                $(TextFieldElement.class).first().getValue());
+    }
+
+    @BeforeAll
+    public static void setupClass() {
+        WebDriverManager.chromedriver().setup();
```

Comment:
Why is this needed? Can we embed it in the base class somehow, so that developers don't need to use this code in their tests?

Reply:
AbstractJUnit5IT - it is an abstract base class already. It is base for Chrome local tests.

Reply:
It is a base class for this demo project -> TestBench users would need to do the same in their projects, can we make their life easier?
And moreover, `WebDriverManager` is a part of `selenium-jupiter`, does the regular TestBench test need it?

Reply:
Also noticed this code in the docs draft:
```
    @BeforeScenario
    public void setUpWebDriver() {
        WebDriverManager.chromedriver().setup();
        driver = TestBench.createDriver(new ChromeDriver());
        driver.get("http://" + IPAddress.findSiteLocalAddress() + ":8080");
        calculator = $(KeypadElement.class).first();
    }
```

It would be better if in `TestBench` method we would call to setup() automatically, or I miss something?

Reply:
I've updated code during work on docs, the point of BDD example is that it does not use `@BrowserTest` feature directly - it is run by JBehave making use of TestBench utility helpers.

Reply:
Thanks for refactoring. I still have a question is `WebDriverManager.chromedriver().setup();` really needed for a regular TestBench test, since `selenium-jupiter` is optional to use, or I missed something?

Reply:
Example updated. Common parts extracted and now BDD tests are managed by TestBench.